src/func.rs

```diff
@@ -150,6 +150,34 @@ impl FuncInstance {
         }
     }
 
+    /// Invoke this function using recycled stacks.
+    ///
+    /// # Errors
+    ///
+    /// Same as [`invoke`].
+    ///
+    /// [`invoke`]: #method.invoke
+    pub fn invoke_with_stack<E: Externals>(
```

Comment:
I considered folding this into `invoke` by adding `stack_recycler: Option<&mut StackRecycler>` as a parameter, but did not do so to avoid breaking the API.